tools/perf/components/perf_test_runner.py

```diff
@@ -178,17 +167,15 @@ def RunSingleTest(cls, profile_dir: Optional[str], binary_path: str,
                        'run_performance_tests.py'))
 
       args.append(f'--benchmarks={benchmark_name}')
-      if out_dir:
-        args.append('--isolated-script-test-output=' +
-                    os.path.join(out_dir, benchmark_name, 'output.json'))
-      else:
-        args.append('--output-format=none')
+      assert (out_dir)
```

Comment:
I think this should be put into `__init__`. also `assert out_dir`, not `(out_dir)`.

Reply:
We can't place it in `__init__` because `out_dir` here can be different.
That because the method is also used for rebasing a profile when `out_dir = None` and `local_run = True`.

Fixed `assert (out_dir)` => `assert out_dir`